{
    "checksum": "e1e56e8be86d226887188cbced3ad9c0e98c9ab865ed2db048ddcc888d80509f",
    "file": "result.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 1,
    "package": "aiomysql",
    "path": "aiomysql/0.0.20-2/.pc/0001-Run-tests-for-python-3.7-in-CI-and-fix-failing-tests.patch/aiomysql/sa/result.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/aiomysql/0.0.20-2/",
        "metric": {
            "size": 868
        },
        "pts_link": "https://tracker.debian.org/pkg/aiomysql",
        "sloc": [
            [
                "python",
                6713
            ],
            [
                "makefile",
                202
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/modules/aiomysql",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/a/aiomysql/0.0.20-2/.pc/0001-Run-tests-for-python-3.7-in-CI-and-fix-failing-tests.patch/aiomysql/sa/result.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 15074,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "0.0.20-2"
}